tutorials/Intro_to_causal_reasoning_with_Dags_bnlearn.Rmd

```diff
@@ -0,0 +1,220 @@
+---
+title: "Introducing bnlearn package in R"
+author: "Sara Taheri"
+date: "5/2/2019"
+header-includes:
+   - \usepackage{amsmath}
+   - \usepackage{soul}
+output: 
+  html_document:
+    self_contained: true
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo=TRUE, warning=FALSE, message=FALSE)
+library(bnlearn)
+library(Rgraphviz)
+```
+
+# Overview of the survey data
+
+## Survey data
+Survey data is a data set that contains information about usage of different transportation systems with a focus on cars and trains for different social groups. It includes these factors:
+
+- **Age (A):** It is recorded as *young* (**young**) for individuals below 30 years, *adult* (**adult**) for individuals between 30 and 60 years old, and *old* (**old**) for people older than 60.
+
+- **Sex (S):** The biological sex of individual, recorded as *male* (**M**) or *female* (**F**).
+
+- **Education (E):** The highest level of education or training completed by the individual, recorded either *high school* (**high**) or *university degree* (**uni**).
+
+- **Occupation (O):** It is recorded as an *employee* (**emp**) or a *self employed* (**self**) worker.
+
+- **Residence (R):** The size of the city the individual lives in, recorded as *small* (**small**) or *big* (**big**).
+
+- **Travel (T):** The means of transport favoured by the individual, recorded as *car* (**car**), *train* (**train**) or *other* (**other**)
+
+Travel is the *target* of the survey, the quantity of interest whose behaviour is under investigation.
+
+```{r, fig.width=3, fig.height=3, fig.align='center'}
+# empty graph
+dag <- empty.graph(nodes = c("A","S","E","O","R","T"))
+arc.set <- matrix(c("A", "E",
+                    "S", "E",
+                    "E", "O",
+                    "E", "R",
+                    "O", "T",
+                    "R", "T"),
+                  byrow = TRUE, ncol = 2,
+                  dimnames = list(NULL, c("from", "to")))
+arcs(dag) <- arc.set
+nodes(dag)
+arcs(dag)
+graphviz.plot(dag)
+```
+
+# Conditional independence in Bayesian networks
+
+Using a DAG structure we can investigate whether a variable is conditionally independent from another variable given a set of variables from the DAG. If the variables
```

Comment:
```suggestion
The DAG is a graphical language for reasoning about conditional independence in the joint probability distribution of the variables in a data generating process.
```

Reply:
After this, add the code examples for V-structure.

Then add Markov blanket.

> **Markov blanket**: The Markov blanket of a node is comprised of its parents, children and children's other parents.  In other words, it is comprised of its parents, children, and any other members of V-structures to which it belongs.

Then include an example of the `mb` code.

For the `mb` we need a plot.  You can use `graphviz` to highlight an MB in your example, or use a free image from the web like this one. ![mb](https://upload.wikimedia.org/wikipedia/commons/e/ea/MarkovBlanket.png)